stl/inc/xmemory

```diff
@@ -1033,147 +1034,190 @@ struct _Iterator_base0 {
 // CLASS _Container_proxy
 struct _Container_base12;
 struct _Container_proxy { // store head of iterator chain and back pointer
-    _Container_proxy() noexcept : _Mycont(nullptr), _Myfirstiter(nullptr) {}
-    _Container_proxy(_Container_base12* _Mycont_) noexcept : _Mycont(_Mycont_), _Myfirstiter(nullptr) {}
+    _CONSTEXPR20 _Container_proxy() noexcept = default;
+    _CONSTEXPR20 _Container_proxy(_Container_base12* _Mycont_) noexcept : _Mycont(_Mycont_) {}
 
     const _Container_base12* _Mycont;
-    _Iterator_base12* _Myfirstiter;
+    mutable _Iterator_base12* _Myfirstiter;

```

Comment:
You aren't initializing these members. Is this intentional?

Reply:
Look a squirrel!

No seriously, that was a stupid oversight

Reply:
And tests finally pass!

Reply:
Who would have thought initializing members could help?